content/images/figures/hetionet_metagraph_subgraph.svg

```diff
@@ -0,0 +1 @@
844654939043158 L 268.33270337962193,-15.549304531002546 L 263.54095262727924,-22.430556214193846 L 262.1056322090329,-19.79619570413357 A 239.49835779022214 239.49835779022214 0 0 0 207.02729672126827,-41.475670426219835"/><text text-anchor="middle" pointer-events="none" font-size="12px" fill="#000000" x="144.56899620298077" y="-43.217976418213084" font-family="sans-serif">UPREGULATES_CuG</text></g></g><g class="layer nodes"><g class="node" transform="translate(54.835332813492585,-71.44168822097942)"><circle class="outline" cx="0" cy="0" r="32.5" fill="#0096FF" stroke="#FFFFFF" stroke-width="1px"/><text text-anchor="middle" pointer-events="none" font-family="sans-serif" dy="2.03125" font-size="32.5" fill="#FFFFFF"/><text text-anchor="middle" pointer-events="none" y="6" font-size="12px" fill="#FFFFFF" font-family="sans-serif"> Gene</text></g><g class="node" transform="translate(254.0614257479432,199.439209858499)"><circle class="outline" cx="0" cy="0" r="32.5" fill="#8E4F00" stroke="#FFFFFF" stroke-width="1px"/><text text-anchor="middle" pointer-events="none" font-family="sans-serif" dy="2.03125" font-size="32.5" fill="#FFFFFF"/><text text-anchor="middle" pointer-events="none" y="6" font-size="12px" fill="#FFFFFF" font-family="sans-serif"> Disease</text></g><g class="node" transform="translate(-105.15100477756428,178.63254867320165)"><circle class="outline" cx="0" cy="0" r="32.5" fill="#FF2600" stroke="#FFFFFF" stroke-width="1px"/><text text-anchor="middle" pointer-events="none" font-family="sans-serif" dy="2.03125" font-size="32.5" fill="#FFFFFF"/><text text-anchor="middle" pointer-events="none" y="6" font-size="12px" fill="#FFFFFF" font-family="sans-serif"> Compound</text></g><g class="node" transform="translate(375.86235093741584,-104.25277094161318)"><circle class="outline" cx="0" cy="0" r="32.5" fill="#008F00" stroke="#FFFFFF" stroke-width="1px"/><text text-anchor="middle" pointer-events="none" font-family="sans-serif" dy="2.03125" font-size="32.5" fill="#FFFFFF"/><text text-anchor="middle" pointer-events="none" y="6" font-size="12px" fill="#FFFFFF" font-family="sans-serif"> Anatomy</text></g><g class="node" transform="translate(458.343297060915,140.77998893983684)"><circle class="outline" cx="0" cy="0" r="32.5" fill="#A5ABB6" stroke="#FFFFFF" stroke-width="1px"/><text text-anchor="middle" pointer-events="none" font-family="sans-serif" dy="2.03125" font-size="32.5" fill="#FFFFFF"/><text text-anchor="middle" pointer-events="none" y="6" font-size="12px" fill="#FFFFFF" font-family="sans-serif"> Symptom</text></g><g class="node" transform="translate(-233.3344148499738,35.601194363297935)"><circle class="outline" cx="0" cy="0" r="32.5" fill="#FFFB00" stroke="#FFFFFF" stroke-width="1px"/><text text-anchor="middle" pointer-events="none" font-family="sans-serif" dy="2.03125" font-size="32.5" fill="#000000"/><text text-anchor="middle" pointer-events="none" y="6" font-size="12px" fill="#000000" font-family="sans-serif"> SideEffect</text></g></g></svg>
```

Comment:
Are some of these bidirectional or undirected relationships? In particular, I'm wondering about things like associates and binds.